ssz/sedes/container.py

```diff
@@ -0,0 +1,74 @@
+from typing import (
+    Any,
+    Dict,
+    Generator,
+    Sequence,
+    Tuple,
+    TypeVar,
+)
+
+from eth_utils import (
+    to_dict,
+)
+from mypy_extensions import (
+    TypedDict,
+)
+
+from ssz.sedes.base import (
+    BaseSedes,
+    LengthPrefixedSedes,
+)
+from ssz.tree_hash.hash_eth2 import (
+    hash_eth2,
+)
+from ssz.utils import (
+    get_duplicates,
+)
+
+AnyTypedDict = TypedDict("AnyTypedDict", {})
+TAnyTypedDict = TypeVar("TAnyTypedDict", bound=AnyTypedDict)
+
+
+class Container(LengthPrefixedSedes[TAnyTypedDict, Dict[str, Any]]):
+
+    length_bytes = 4
+
+    def __init__(self, fields: Sequence[Tuple[str, BaseSedes[Any, Any]]]) -> None:
+        field_names = [field_name for field_name, field_sedes in fields]
```

Comment:
This will result in `self.fields` being mutable which doesn't seem desired.  Convert to `tuple`?